pkg/channel/consolidated/dispatcher/dispatcher.go

```diff
@@ -328,7 +328,7 @@ func (d *KafkaDispatcher) subscribe(channelRef eventingchannels.ChannelReference
 
 	handler := &consumerMessageHandler{d.logger, sub, d.dispatcher}
 
-	consumerGroup, err := d.kafkaConsumerFactory.StartConsumerGroup(groupID, []string{topicName}, d.logger, handler)
+	consumerGroup, err := d.kafkaConsumerFactory.StartConsumerGroup(context.Background(), groupID, []string{topicName}, d.logger, handler)
```

Comment:
Can we use a "real" context?

Reply:
yes we should. I'm opening an issue to track it as I don't know enough the dispatcher to fix this.

Reply:
here: https://github.com/knative-sandbox/eventing-kafka/issues/105